Makefile

```diff
@@ -82,3 +82,8 @@ release: check-api-token check-app deps-vendor-cli lint
 		--release-notes $(release_notes) \
 		--ensure-channel
 
+.PHONY: kurl
+kurl: check-api-token check-app deps-vendor-cli lint
+	deps/replicated installer create \
+		--app ${app_slug} \
+		--yaml-file kurl-installer.yaml
```

Comment:
why not use --auto?

Reply:
Good point! This is probably because I was not aware of the flag at that time. Let me test and change if it works.